client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -138,26 +140,59 @@ export function getEnhancedTasks(
 					};
 					break;
 				case 'plan_selected':
+					/* eslint-disable no-case-declarations */
+					const openPlansPage = () => {
+						recordTaskClickTracksEvent( flow, task.completed, task.id );
+						if ( displayGlobalStylesWarning ) {
+							recordTracksEvent(
+								'calypso_launchpad_global_styles_gating_plan_selected_task_clicked'
```

Comment:
Nitpick: can we add the flow as an extra prop? I've seen a significant bump in clicks and it would be useful to see which flow is generating these clicks.